functions.php

```diff
@@ -337,3 +337,25 @@ function twentysixteen_hex2rgb( $color ) {
  * Customizer additions.
  */
 require get_template_directory() . '/inc/customizer.php';
+
+/*
+ * Add image size to enhance responsive image functionality
+ *
+ * @link https://developer.wordpress.org/reference/functions/add_image_size/
+ */
+add_image_size( 'max-content', 600, 600 );
+
+/*
+ * Add custom image sizes attribute to enhance responsive image functionality
+ *
+ * @link https://core.trac.wordpress.org/browser/trunk/src/wp-includes/media.php#L1035
+ */
+add_filter( 'wp_image_sizes_args', 'twentysixteen_image_sizes_attr', 10 , 3 );
```

Comment:
It's minor but could you move this after the function for the sake of consistency?